### PR TITLE
Add descriptions to the tasks data API

### PR DESCRIPTION
**Pull request by zhiltsov-max**
### Motivation and context

- Added descriptions for the server task data API fields
- Fixed docs

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/basics/create_an_annotation_task.md [hunk 1]
```diff
@@ -263,6 +263,12 @@ The following parameters are available:
...
 
 <!--lint enable maximum-line-length-->
+
+To save and open the task click **Submit & Open** .
```
Comment (SpecLad) [inline site/content/en/docs/manual/basics/create_an_annotation_task.md:270]:
```suggestion
To save and open the task, click **Submit & Open** .
```

File: site/content/en/docs/manual/basics/create_an_annotation_task.md [hunk 2]
```diff
@@ -263,6 +263,12 @@ The following parameters are available:
...
+
+To create several tasks in sequence, click on the **Submit & Continue**.
```
Comment (SpecLad) [inline site/content/en/docs/manual/basics/create_an_annotation_task.md:272]:
```suggestion
To create several tasks in sequence, click **Submit & Continue**.
```

**Review by SpecLad — CHANGES_REQUESTED** [review]

File: cvat/apps/engine/serializers.py [hunk 3]
```diff
@@ -680,15 +678,46 @@ def __init__(self, *args, **kwargs):
...
+
+    image_quality = serializers.IntegerField(min_value=0, max_value=100,
+        help_text="Image quality to use during annotation")
+    use_zip_chunks = serializers.BooleanField(default=False,
+        help_text=textwrap.dedent("""\
+            Force the use of frame-based chunks for videos.
+            By default, chunks are created from video segments
+        """))
+    client_files = ClientFileSerializer(many=True, default=[],
+        help_text="Files to be uploaded")
```
[the remark this answers is not in the document]
Comment (zhiltsov-max) [reply cvat/apps/engine/serializers.py:694]: Probably, all fields must be marked write-only

File: cvat/apps/engine/serializers.py [hunk 4]
```diff
@@ -680,15 +678,46 @@ def __init__(self, *args, **kwargs):
...
+    copy_data = serializers.BooleanField(default=False, help_text=textwrap.dedent("""\
+            Allows to copy data from the server file share to CVAT during the task creation.
+            This will create a copy of the data, making the server independent from
+            the file share availability
+        """))
+    cloud_storage_id = serializers.IntegerField(write_only=True, allow_null=True, required=False,
+        help_text=textwrap.dedent("""\
+            Allows to specify a cloud storage to be used as the data source.
+            The cloud storage record needs to be defined first. Both user
+            sandbox and organization cloud storages can be used
```
[the remark this answers is not in the document]
Comment (zhiltsov-max) [reply cvat/apps/engine/serializers.py:713]: > Both user sandbox and organization cloud storages can be used

@Marishka17, is it true at all? I suspect it's not supposed.

File: site/content/en/docs/manual/advanced/data_on_fly.md [hunk 5]
```diff
@@ -22,9 +22,13 @@ This method of working with data allows:
 - reduce the task creation time.
 - store data in a cache of the limited size with a policy of evicting less popular items.
 
-Unfortunately, this method will not work for all videos with a valid manifest file. If there are not enough keyframes
-in the video for smooth video decoding, the task will be created in another way. Namely, all chunks will be prepared
-during task creation, which may take some time.
+Unfortunately, this method has several drawbacks:
+- The first access to the data will take more time
```
[the remark this answers is not in the document]
Comment (SpecLad) [reply site/content/en/docs/manual/advanced/data_on_fly.md:26]: Ping? I think you missed this one.

**Pull request merged**